components/FeatureHighlight/tests/unit/FeatureHighlightTitleBodyAccessibilityMutatorTests.m

```diff
@@ -89,9 +89,9 @@ - (void)testMutatorChangesTextColorToExpectedColor {
   // Hard coded values, if this test fails it only means that we could mean that we changed the
   // accessible color we are returning.
   XCTAssertEqualObjects(featureHighlightViewController.titleColor,
-                        [UIColor colorWithWhite:0 alpha:0.87000000476837158]);
+                        [UIColor colorWithWhite:0 alpha:(CGFloat)0.87]);
   XCTAssertEqualObjects(featureHighlightViewController.bodyColor,
-                        [UIColor colorWithWhite:0 alpha:0.54000002145767212]);
+                        [UIColor colorWithWhite:0 alpha:(CGFloat)0.54]);
```

Comment:
is there a reason why those numbers were so insanely precise before?

Reply:
I believe if you pass `0.54f`, `(float)0.54`, and `(CGFloat)0.54000002145767212` you get the same value. However, if you pass `(CGFloat)0.54` you get a value closer to 0.54. Since the colors previously used `0.54f`, I believe the test code just passed a double that resulted in the correct value.

```objc
  CGFloat singleWidth = (float)0.54;
  CGFloat doubleWidthLong = (CGFloat)0.54000002145767212;
  CGFloat doubleWidthShort = (CGFloat)0.54;
  NSLog(@"\n S: %.024f\nDl: %.024f\nDs: %.024f\n", singleWidth, doubleWidthLong, doubleWidthShort);

/*
 S: 0.540000021457672119140625
Dl: 0.540000021457672119140625
Ds: 0.540000000000000035527137
*/
```